src/components/Wallets/Wallet.vue

```diff
@@ -337,7 +374,8 @@ export default {
             privateKey,
             assetId: this.wallet.assetId,
             to: this.transferForm.to,
-            amount: this.transferForm.amount
+            description: this.transferForm.description,
```

Comment:
It seems to throw an error if the description is empty because the initial value of `description` is not an empty string but null.